src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3613,24 +3613,18 @@
      * <p>parses as:</p>
      *
      * <pre>
-     * +--VARIABLE_DEF
-     *     |
-     *     +--MODIFIERS
-     *         |
-     *         +--LITERAL_PUBLIC (public)
-     *     +--TYPE
-     *         |
-     *         +--IDENT (Collection)
-     *             |
-     *             +--TYPE_ARGUMENTS
-     *                 |
-     *                 +--GENERIC_START (&lt;)
-     *                 +--TYPE_ARGUMENT
-     *                     |
-     *                     +--WILDCARD_TYPE (?)
-     *                 +--GENERIC_END (&gt;)
-     *     +--IDENT (a)
-     *     +--SEMI (;)
+     * --VARIABLE_DEF -&gt; VARIABLE_DEF
```

Comment:
please remove leading "--" it will indent whole code to left to make it less wide